From 095797f03cacc1103894050accd847eb61c32a60 Mon Sep 17 00:00:00 2001
From: Thang Tran <thuutran@amperecomputing.com>
Date: Fri, 17 Jan 2025 13:30:42 +0700
Subject: [PATCH] mtjeffeson: correct the thresholds/EntityID of sensors

Update the thresholds of sensors 1V8_CPLD_AUX, 5V_MAIN, S0_1V8_RC5A and
S0_1V8_SOC.

update the EntityInstance/EntityId of sensors NVMeFBP_MaxTemp,
NVMeM2_MaxTemp, FrontBP_MaxTemp, FAN0, FAN1, FAN2 and FAN3.

Upstream-Status: Pending

Signed-off-by: Thang Tran <thuutran@amperecomputing.com>
---
 configurations/mtjefferson_mb.json | 32 ++++++++++++++++++++----------
 1 file changed, 22 insertions(+), 10 deletions(-)

diff --git a/configurations/mtjefferson_mb.json b/configurations/mtjefferson_mb.json
index a14ed9d..1c58f73 100644
--- a/configurations/mtjefferson_mb.json
+++ b/configurations/mtjefferson_mb.json
@@ -417,15 +417,15 @@
                     },
                     {
                         "Direction": "less than",
-                        "Name": "lower critical",
-                        "Severity": 1,
+                        "Name": "lower non critical",
+                        "Severity": 0,
                         "Value": 1.53
                     },
                     {
                         "Direction": "less than",
-                        "Name": "lower non critical",
-                        "Severity": 0,
-                        "Value": 1.434
+                        "Name": "lower critical",
+                        "Severity": 1,
+                        "Value": 1.435
                     }
                 ],
                 "Type": "ADC"
@@ -445,7 +445,7 @@
                         "Direction": "greater than",
                         "Name": "upper critical",
                         "Severity": 1,
-                        "Value": 6.022
+                        "Value": 6.023
                     },
                     {
                         "Direction": "greater than",
@@ -463,7 +463,7 @@
                         "Direction": "less than",
                         "Name": "lower non critical",
                         "Severity": 0,
-                        "Value": 4.262
+                        "Value": 4.263
                     }
                 ],
                 "Type": "ADC"
@@ -817,7 +817,7 @@
                         "Direction": "less than",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 1.434
+                        "Value": 1.435
                     },
                     {
                         "Direction": "less than",
@@ -977,7 +977,7 @@
                         "Direction": "less than",
                         "Name": "lower critical",
                         "Severity": 1,
-                        "Value": 1.434
+                        "Value": 1.435
                     },
                     {
                         "Direction": "less than",
@@ -1121,6 +1121,8 @@
                 "Type": "ChassisIntrusionSensor"
             },
             {
+                "EntityId": 29,
+                "EntityInstance": 1,
                 "Address": "0x5c",
                 "Bus": "8",
                 "Connector": {
@@ -1151,6 +1153,8 @@
                 "Type": "I2CFan"
             },
             {
+                "EntityId": 29,
+                "EntityInstance": 3,
                 "Address": "0x5c",
                 "Bus": "8",
                 "Connector": {
@@ -1181,6 +1185,8 @@
                 "Type": "I2CFan"
             },
             {
+                "EntityId": 29,
+                "EntityInstance": 5,
                 "Address": "0x5c",
                 "Bus": "8",
                 "Connector": {
@@ -1211,6 +1217,8 @@
                 "Type": "I2CFan"
             },
             {
+                "EntityId": 29,
+                "EntityInstance": 7,
                 "Address": "0x5c",
                 "Bus": "8",
                 "Connector": {
@@ -1245,7 +1253,7 @@
                 "Type": "MB Upstream Port"
             },
             {
-                "EntityId": 30,
+                "EntityId": 4,
                 "EntityInstance": 1,
                 "MaxValue": 127,
                 "MinValue": 0,
@@ -1349,6 +1357,8 @@
                 "Units": "DegreesC"
             },
             {
+                "EntityId": 15,
+                "EntityInstance": 1,
                 "MaxValue": 127,
                 "MinValue": 0,
                 "Name": "FrontBP_MaxTemp",
@@ -1368,6 +1378,8 @@
                 "Units": "DegreesC"
             },
             {
+                "EntityId": 4,
+                "EntityInstance": 2,
                 "MaxValue": 127,
                 "MinValue": 0,
                 "Name": "NVMeFBP_MaxTemp",
-- 
2.43.0

